docs/reference/authentication.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Authentication 
+description: This section covers Authentication of your lakeFS server.
+parent: Reference
+nav_order: 59
+has_children: false
+---
+
+# Authentication 
+
+{: .no_toc }
+
+{% include toc.html %}
+
+## Authentication
+
+### User Authentication
+
+lakeFS authenticates users from a built-in authentication database, or
+optionally from a configured LDAP server.
+
+#### Built-in database
+
+The built-in authentication database is always present and active.  Use the
+Web UI at Administration / Users to create users.  Users have an access key
+`AKIA...` and an associated secret access key.  These credentials are valid
```

Comment:
Don't we have email authentication as well now?

Reply:
We do, The idea here is to add exactly that (in a coming PR hopefully by today/tomorrow), and as a first step we felt is made more sense to split the authentication and authorization into two separate docs.